_episodes/04-docker-hub.md

```diff
@@ -23,11 +23,11 @@ and maybe `busybox`. Where did these containers come from?  The Docker Hub!
 
 ## Introducing the Docker Hub
 
-The Docker Hub is an online repository of container images, a vast number of which are publicly available. A large number of the images are curated by the developers of the software that they package. Also, many commonly used pieces of software that have been containerised into images are specifically endorsed, which means that you can trust the containers to have been checked for functionality, stability, and that they don't contain malware.
+The Docker Hub is an online repository of container images, a vast number of which are publicly available. A large number of the images are curated by the developers of the software that they package. Also, many commonly used pieces of software that have been containerized into images are specifically endorsed, which means that you can trust the containers to have been checked for functionality, stability, and that they don't contain malware.
```

Comment:
Rather than "specifically endorsed", should we say "officially endorsed"?
```suggestion
The Docker Hub is an online repository of container images, a vast number of which are publicly available. A large number of the images are curated by the developers of the software that they package. Also, many commonly used pieces of software that have been containerized into images are officially endorsed, which means that you can trust the containers to have been checked for functionality, stability, and that they don't contain malware.
```